src/sagas/nanoContract.js

```diff
@@ -117,14 +116,13 @@ export function* registerNanoContract({ payload }) {
   const nc = {
     address,
     ncId,
-    blueprintId: ncState.blueprint_id,
```

Comment:
Why did you remove this? The method `wallet.storage.registerNanoContract` expects a `INcData` as parameter, which has the `bluerprintId` attribute.

Reply:
Yes, I know, but the method expects this parameter, we must send it. In the desktop wallet I used another request to get this information,

Reply:
Added again.